lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -27,6 +28,8 @@ module Cardano.Wallet.Shelley.Network.Blockfrost
 
     -- * Internal
     , getPoolPerformanceEstimate
+    , getStakePoolsSummary'
```

Comment:
I don't see how this internal function is used inside this module. 
We have some functionality to convert (`class FromBlockfrost`) and run (`runBlockfrost`) the blockfrost computation to `IO` monad already. It also includes error handling.
Is there a reason not to use the uniform approach when running BF computations?

Reply:
My intention is to eventually set
```
stakeDistribution = \_ -> runBlockfrost getStakePoolsSummary'
```
in `withNetworkLayer`, but due to issues with backward compatibility of the stake pool rankings, I can't do that right now. I had to export the function `getStakePoolsSummary'` in order to appease the linter.

The `getStakePoolsSummary'` function interleaves several queries in order to obtain the desired result — this results in some interleaving between type conversion and monadic actions, the `FromBlockfrost` class doesn't quite fit the entire bill. Still, the function is intended to be an argument to the `runBlockfrost` function, though the seeming generality of the monad type `BF.MonadBlockfrost m => m` is a bit confusing.

That said, the code was written before the `FromBlockfrost` class existed. I'll revisit its uses.